src/recipe.cpp

```diff
@@ -489,6 +536,67 @@ bool recipe::has_byproducts() const
     return !byproducts.empty();
 }
 
+std::string recipe::required_proficiencies_string( const Character &c ) const
+{
+    std::vector<proficiency_id> required_profs;
+    std::vector<std::pair<proficiency_id, float>> used_profs;
+
+    for( const recipe_proficiency &rec : proficiencies ) {
+        if( rec.required ) {
+            required_profs.push_back( rec.id );
+        } else {
+            used_profs.push_back( { rec.id, rec.time_multiplier } );
+        }
+    }
+    std::string required = enumerate_as_string( required_profs.begin(),
+    required_profs.end(), [&]( const proficiency_id & id ) {
+        const std::string color = c.has_proficiency( id ) ? "green" : "red";
+        return string_format( "<color_%s>%s</color>", color, id->name() );
+    } );
+
+    std::string used = enumerate_as_string( used_profs.begin(),
+    used_profs.end(), [&]( const std::pair<proficiency_id, float> &pair ) {
+        const std::string color = c.has_proficiency( pair.first ) ? "white" : "yellow";
+        return string_format( "<color_cyan>%s</color> <color_%s>%gx</color>", pair.first->name(), color,
+                              pair.second );
+    } );
+    used = string_format( "Proficiencies Used: %s", used );
```

Comment:
Needs to be translatable - see "doc/TRANSLATING.md".

Reply:
Thanks for the review! I've made changes as per your suggestions.